yt_dlp/downloader/hls.py

```diff
@@ -29,7 +29,7 @@ class HlsFD(FragmentFD):
     FD_NAME = 'hlsnative'
 
     @staticmethod
-    def can_download(manifest, info_dict, allow_unplayable_formats=False, with_crypto=can_decrypt_frag):
+    def can_download(manifest, info_dict, allow_unplayable_formats=False, with_crypto=True):
```

Comment:
This is annoying, should be fixed.
I just hoping I didn't miss other usage of removed flags.